web-bluetooth/service-worker.js

```diff
@@ -1,6 +1,6 @@
 function addToCache(request, networkResponse) {
   return caches.open('web-bluetooth')
-    .then(cache => cache.put(request, networkResponse.clone()));
+    .then(cache => cache.put(request, networkResponse));
```

Comment:
This PR moved the `clone()` out of `addToCache()`, yes. So I think it's already doing what you're suggesting.

You might be confused based on the partial diffs?